Failed test in devel/milestone due to drop support of python 3.11

Ansible [drops support for Python 3.11](https://github.com/ansible/ansible/pull/85590) recently that caused failed tests:

{code}
Run python3 -m pip install https://github.com/ansible/ansible/archive/devel.tar.gz --disable-pip-version-check
Collecting https://github.com/ansible/ansible/archive/devel.tar.gz
  Downloading https://github.com/ansible/ansible/archive/devel.tar.gz (3.3 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.3/3.3 MB 107.5 MB/s eta 0:00:00
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
  Preparing metadata (pyproject.toml): started
  Preparing metadata (pyproject.toml): finished with status 'done'
INFO: pip is looking at multiple versions of ansible-core to determine which version is compatible with other requirements. This could take a while.
ERROR: Package 'ansible-core' requires a different Python: 3.11.13 not in '>=3.12'
Error: Process completed with exit code 1.
{code}

So, this change should be reflected in matrix

--- a/.github/workflows/integration_simple.yml
+++ b/.github/workflows/integration_simple.yml
@@ -15,13 +15,21 @@ on:
         # 2.19 supports Python 3.11-3.13
         # support for Python 3.13 added and 3.10 removed in 2.18 for control node
         # target node supported Python 3.8-3.13 as of 2.18 and 2.19
-        # milestone is and devel is switched to 2.20
+        # milestone is and devel is switched to 2.20 and drops support for Python 3.11
         # https://docs.ansible.com/ansible/devel/roadmap/ROADMAP_2_18.html
         default: >-
           [
             {
               "ansible-version": "devel",
+              "python-version": "3.11"
+            },
+            {
+              "ansible-version": "devel",
               "python-version": "3.10"
+            },
+            {
+              "ansible-version": "milestone",
+              "python-version": "3.11"
             },
             {
               "ansible-version": "milestone",

--- a/.github/workflows/sanity.yml
+++ b/.github/workflows/sanity.yml
@@ -15,13 +15,21 @@ on:
         # 2.19 supports Python 3.11-3.13
         # support for Python 3.13 added and 3.10 removed in 2.18 for control node
         # target node supported Python 3.8-3.13 as of 2.18 and 2.19
-        # milestone is and devel is switched to 2.20
+        # milestone is and devel is switched to 2.20 and drops support for Python 3.11
         # https://docs.ansible.com/ansible/devel/roadmap/ROADMAP_2_19.html
         default: >-
           [
             {
               "ansible-version": "devel",
+              "python-version": "3.11"
+            },
+            {
+              "ansible-version": "devel",
               "python-version": "3.10"
+            },
+            {
+              "ansible-version": "milestone",
+              "python-version": "3.11"
             },
             {
               "ansible-version": "milestone",

--- a/.github/workflows/unit_galaxy.yml
+++ b/.github/workflows/unit_galaxy.yml
@@ -15,13 +15,21 @@ on:
         # 2.19 supports Python 3.11-3.13
         # support for Python 3.13 added and 3.10 removed in 2.18 for control node
         # target node supported Python 3.8-3.13 as of 2.18 and 2.19
-        # milestone is and devel is switched to 2.20
+        # milestone is and devel is switched to 2.20 and drops support for Python 3.11
         # https://docs.ansible.com/ansible/devel/roadmap/ROADMAP_2_19.html
         default: >-
           [
             {
               "ansible-version": "devel",
+              "python-version": "3.11"
+            },
+            {
+              "ansible-version": "devel",
               "python-version": "3.10"
+            },
+            {
+              "ansible-version": "milestone",
+              "python-version": "3.11"
             },
             {
               "ansible-version": "milestone",

--- a/.github/workflows/unit_source.yml
+++ b/.github/workflows/unit_source.yml
@@ -15,10 +15,14 @@ on:
         # 2.19 supports Python 3.11-3.13
         # support for Python 3.13 added and 3.10 removed in 2.18 for control node
         # target node supported Python 3.8-3.13 as of 2.18 and 2.19
-        # milestone is and devel is switched to 2.29
+        # milestone is and devel is switched to 2.20 and drops support for Python 3.11
         # https://docs.ansible.com/ansible/devel/roadmap/ROADMAP_2_18.html
         default: >-
           [
+            {
+              "ansible-version": "milestone",
+              "python-version": "3.11"
+            },
             {
               "ansible-version": "devel",
               "python-version": "3.11"

